devoxx-2017-be/cfp-impl/src/main/java/org/tweetwall/devoxx/api/cfp/client/impl2017be/CFPClientDevoxx2017BE.java

```diff
@@ -49,68 +49,82 @@
     private static final String CONFERENCE_BASE_URI = "https://cfp.devoxx.be/api/conferences/DVBE17";
 
     @Override
-    public Events getEvents() {
-        return readFrom(getResponse(BASE_URI + "conferences/"), Events.class);
+    public Optional<Events> getEvents() {
+        return getOptionalResponse(BASE_URI + "conferences/")
```

Comment:
Maybe this could aggregated into one method call (uri, type) instead of repeating the flatMap code. This is a generic comment applicable to all usages of this pattern.